packages/patternfly-3/patternfly-react-extensions/src/components/ClassificationBanner/ClassificationBanner.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * ClassificationBanner Component for PatternFly React
+ */
+class ClassificationBanner extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      closed: props.closed || false
+    }
+
+  }
+
+  onClose = ()=>{
+    this.setState({closed:true})
+  }
+
+  componentWillReceiveProps(nextProps) {
+    
+    if (nextProps.closed !== this.state.closed) {
+      this.setState({ closed: nextProps.closed});
+    }
+  }
+
+  render(){
+    const { 
+      children, 
+      bottomBanner,
+      hostName,
+      userName,
+      classificationLevel,
+      backgroundColor,
+      closeButton,
+      userNamePosition,
+      hostNamePosition,
+      bannerColor,
+      ...props } = this.props;
+    
+    const levelNameAppendix = classificationLevel==='999'?'':(classificationLevel==='0'?'-unclassified':(classificationLevel==='1'?'-classified':'-proprietary-level'));
+
+    const classificationBannerClasses = {
+      top: classNames('classification-banner-pf-banner',
+      'classification-banner-pf-banner-top',
+      {
+        'classification-banner-pf-hide': this.state.closed,
+      },
+      'classification-banner-pf-banner'+levelNameAppendix),
+      bottom: classNames(
+      'classification-banner-pf-banner',
+      'classification-banner-pf-banner'+levelNameAppendix,
+      {
+      'classification-banner-pf-hide':(!bottomBanner)||this.state.closed
+      },
+      'classification-banner-pf-banner-bottom'),
+      closeButton: classNames({
+        'classification-banner-pf-close pficon-error-circle-o': closeButton
+      }),
+      children: classNames({
+        'classification-banner-pf-children-no-bottom':(!this.state.closed && !bottomBanner),
+        'classification-banner-pf-children':(!this.state.closed && bottomBanner)})
+    }
+    const levelName = classificationLevel==='999'?'':(classificationLevel==='0'?'Unclassified':(classificationLevel==='1'?'Classified':(classificationLevel==='2'?'Proprietary Level 1':'Proprietary Level 2')));
```

Comment:
@zeyap these values probably should not be defined as the text should be configurable.